Code/Editor/EditorViewportCamera.h

```diff
@@ -86,21 +87,23 @@ namespace SandboxEditor
     //! @param position The new position of the camera in world space.
     //! @param pitch Amount of pitch in radians.
     //! @param yaw Amount of yaw in radians.
-    SANDBOX_API void HandleViewportCameraTransitionFromSetting(AzFramework::ViewportId viewportId, const AZ::Vector3& position, float pitch, float yaw);
+    SANDBOX_API void HandleViewportCameraTransitionFromSetting(
+        AzFramework::ViewportId viewportId, const AZ::Vector3& position, float pitch, float yaw, float duration);
 
     //! Will call either Set or Interpolate camera transform depending on user setting.
     //! @param position The new position of the camera in world space.
     //! @param pitch Amount of pitch in radians.
     //! @param yaw Amount of yaw in radians.
-    SANDBOX_API void HandleDefaultViewportCameraTransitionFromSetting(const AZ::Vector3& position, float pitch, float yaw);
+    SANDBOX_API void HandleDefaultViewportCameraTransitionFromSetting(const AZ::Vector3& position, float pitch, float yaw, float duration);
```

Comment:
Suggestion: Since it seems like all the uses of this API consist of passing in the value read in from the settings, maybe the API should internally look up that setting instead of exposing duration.

Reply:
Yeah that's not a bad plan, keeps that interface a little simpler, I'll remove it.